### ISSUE TITLE
X# compliler flags -xpp1 -vo16 isn't work (dialect XBase++)

### ISSUE THREAD

**Issue by DenGhostYY** [X-Sharp/XSharpPublic#1243]
**Describe the bug**
X# compliler flags -xpp1 -vo16 isn't work

**To Reproduce bug of flag -xpp1**
```
procedure main()
    local o
    
    o := DerivedExample():new()
    
    ? o:ClassName()
    ? o:IsDerivedFrom("Example")
    return

class DerivedExample from Example
endclass

class Example
endclass
```

**Expected behavior (xBase++ exe)**
Output
```
DerivedExample
Y
```

**Actual behavior (X# exe)**
```
Unhandled Exception XSharp.Error: No exported method
   в XSharp.RT.OOPHelpers.DoSend(Object oObject, String cMethod, __Usual[] args, String cCaller)
   в XSharp.RT.Functions.__InternalSend(__Usual oObject, String cMethod, __Usual[] args)
   в xsharp3.Exe.Functions.Main(__Usual[] Xs$Args)
   в xsharp3.Exe.Functions.Start(String[] Xs$Args)
```

**To Reproduce bug of flag -vo16**
```
procedure main()
    local o
    
    o := DerivedClass():new(3, 4)
  
    ? o:GetA()
    ? o:GetB()
    return

class DerivedClass from BaseClass
endclass

class BaseClass
exported:
    inline method Init(a, b)
        ::_a := a
        ::_b := b
    return self
  
    inline method GetA(); return ::_a
    inline method GetB(); return ::_b
  
hidden:
    var _a
    var _b
endclass
```
**Expected behavior (xBase++ exe)**
Output
```
3
4
```

**Actual behavior (X# compiler)**
```
Error	XS1729	'DerivedClass' does not contain a constructor that takes 2 arguments
Warning	XS9201	XPP The 'FROM' clause is interpreted as SHARING because variables in parent classes are always shared in the .Net Runtime.
```

**Additional context**
X# Compiler version 2.14.0.4 (release)
xsc.exe -dialect:xBase++ -xpp1 -lb -memvar -vo1 -vo5 -vo10 -vo15 -vo16

**Comment by RobertvanderHulst** [comment]
With the current X# compiler (2.16) the first example returns 
```
DerivedExample
.T.
```
when /xpp1 is enabled.

**Comment by RobertvanderHulst** [comment]
I see that indeed the second example fails to compile because there is no constructor declared for DerivedClass , and even when the compiler option Create Clipper constructors (/vo16) is enabled it does not generate the constructor.
For now this can be solved by adding this to the Derived Class:
```

exported:
    inline method Init(a, b)
    super(a,b)
```
I will fix this for 2.16

**Comment by cpyrgas** [comment]
Confirmed fixed

**Comment by cpyrgas** [comment]
Indeed problem #1 now fails in 2.19 and 2.20

**Comment by cpyrgas** [comment]
All confirmed fixed for 2.20
